diagrams/request_uri_mode_create.md

```diff
@@ -0,0 +1,49 @@
+```plantuml
+@startuml
+
+autonumber
+
+box "Wallet"
+participant "Metadata" as wm
+participant "Authorization Endpoint" as w
+end box
+
+participant "User Agent" as u
+
+box "Verifier"
+participant "Frontend" as r
+participant "Request Endpoint" as rp
+participant "Response Endpoint" as rb
+end box
+
+u --> r : use
+activate r
+
+r --> u: **signed authorization request**\n(client_id, request_uri, request_uri_mode=create, state)
+deactivate r
+u --> w: **signed authorization request**\n(client_id, request_uri, request_uri_mode=create, state)
+activate w
+w --> w: check authorization request signature
+w --> w: check on trustworthiness of Verifier (approach on trust mechanism)
+w --> rp: POST **request_uri** (\n[OPTIONALstate,\n[OPTIONAL]issuer, \n[OPTIONAL]wallet_metadata, \n[OPTIONAL]w_nonce, \n[OPTIONAL]w_ephm_key)
+rp --> wm: [OPTIONAL] get wallet metadata
+wm --> rp: [OPTIONAL] wallet metadata
+rp -> rp: create and sign (and optionally encrypt) presentation request object (client_id, w_nonce, nonce, response_uri, \npresentation_definition, state)
+rp --> w: **signed (optionally encrypted) request object** (client_id, w_nonce, nonce, response_uri, \npresentation_definition, state)
+note over u, w: do we want to allow unsigned presentation request objects, too?
```

Comment:
9. To this question, imo, we should allow unsigned Authz Request objects but then the same rules as in JAR apply which is setting a dedicated parameter `require_signed_request_object` to `false` in the Authz Request.

Reply:
require_signed_request_object is a metadata parameter - are you suggesting the wallet should have that in its metadata, set to false? I think it is a bit tricky since we use JAR twice, for the request and the request_uri.